spec/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```

Comment:
Why is this file here? This seems like something that should only be on gh-pages. Also, redirects like this can be generated automatically with sphinx-reredirects.

Reply:
To be copied to `gh-pages` from the `make build` command. As mentioned in OP, I want to keep the build process fully reproducible. Need to explore `sphinx-reredirects` generally, but not applicable for this file as it exists outside a Sphinx doc (i.e. `data-apis.org/array-api/index.html`, not `data-apis.org/array-api/{2021.12/latest/draft}/index.html`. Again, something for a `spec/README.me` as you say.